Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.cpp

```diff
@@ -1529,6 +1534,12 @@ bool ApplicationManagerBase::Activate()
         {
             AssetProcessor::BuilderRef builder;
             AssetProcessor::BuilderManagerBus::BroadcastResult(builder, &AssetProcessor::BuilderManagerBus::Events::GetBuilder, true);
+
+            if (!builder)
+            {
+                AZ_Error("ApplicationManagerBase", false, "AssetBuilder process failed to start.  Builder registration cannot complete.  Shutting down.");
```

Comment:
Do we know anything about the builder that may aid troubleshooting?

Reply:
In the end this message wasn't super necessary because my other changes fixed the existing error messages not showing up, but this does add the additional context of the builder being used for registration and that the failure will cause a shutdown.  There are 2 other messages that show up before this indicating the builder has timed out and failed to start because of that.  So it should give the user the needed info.